css/theme/black.konstantina.css

```diff
@@ -0,0 +1,294 @@
+/**
```

Comment:
we don't need to copy the whole file, just add a new file with our custom changes. I can take care of that